modules/serverless-create-func-kn.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-functions-developer-guide.adoc
+
+[id="serverless-create-func-kn_{context}"]
+= Creating functions
+
+The following procedure describes how you can create a basic serverless function using the `kn` CLI.
```

Comment:
I would remove the first few words of this opening sentence:

`You can create a basic serverless function using the `kn` CLI.`